rest-server/core/src/main/scala/com/actionml/core/jobs/JobManager.scala

```diff
@@ -37,10 +38,19 @@ import scala.util.control.NonFatal
 
 
 trait JobManagerInterface {
-  def addJob(engineId: String, f: Future[_] = Future.successful(()), c: Cancellable = Cancellable.noop, comment: String = ""): JobDescription
-  def getActiveJobDescriptions(engineId: String): Map[String, JobDescription]
+  def addJob(
+        engineId: String,
+        f: Future[_] = Future.successful(()),
+        c: Cancellable = Cancellable.noop,
+        comment: String = "",
+        initStatus: JobStatus = JobStatuses.queued): JobDescription
```

Comment:
incorrect indent, according to https://docs.scala-lang.org/style/indentation.html